Forest Brawler becomes Forest Reaver

ghkiha opened the pull request:
<!-- Write **BELOW** The Headers and **ABOVE** The comments else it may not be viewable. -->
<!-- Note: PRs including balance changes authored by anyone other than maintainers and official devs will not be considered. -->

## About The Pull Request

<!-- Describe The Pull Request. Please be sure every change is documented or this can delay review or prevent the PR from being merged! -->This shifts the widely controversial subclass of forest brawler to be a more identity driven variant of forest guard. Their skills shift from being a pugilist to a dual wielding and or two-handed axe/mace wielder with a bit more stamina and a bit less speed than the infantry subclass, while still having the ability to dodge as they don't focus on defense, but rather a berserker aesthetic. Wrestling is now equal to the infantry, at tier 3 instead of 4.
Summary
- Lowered wrestling to equal infantry
- Gave Warden actual combat axe
- Gave Reaver woodcutter axe
- Changed Brawler to Reaver, replacing Dodge Expert from Brawler to No Pain Stun
- Swapped three strength to three endurance and made subclass malus for speed to neutral instead, leaving it still the slowest subclass.

## Why It's Good For The Game

<!-- Please add a short description of why you think these changes would benefit the game. If you can't justify it in words, it might not be worth adding. -->A grizzled old berserker is a better fit to the PTSD ridden barracks of the Forest Guard than some guy who is just a monk but better. Has more identity, and is more feasible to facing threats without solely focusing on wrestling.

## Pre-Merge Checklist
<!-- Don't bother filling these in while creating your Pull Request, just click the checkboxes after the Pull Request is opened and you are redirected to the page. -->
- [x] You tested this on a local server.
- [x] This code did not runtime during testing.
- [x] You documented all of your changes.
<!-- Neither the compiler nor workflow checks are perfect at detecting runtimes and errors. It is important to test your code/feature/fix locally. -->

assortedbeads commented:
Got a few issues with this.

- Dual-Wielding dont do anything tangible, and giving them 4 axe maces then lowering their Unarmed Wrestling just makes them an off colour infantry.
- They have a different niche from the monk, much slower and less skilled but more armored.
- Not sure anyone actually Asked for this change. I play forest brawler on occasion and I enjoy the class, if I wanted to dual wield I would play infantry

Draggeru commented:
> Got a few issues with this.
> 
> * Not sure anyone actually Asked for this change. I play forest brawler on occasion and I enjoy the class, if I wanted to dual wield I would play infantry

The host has, actually

assortedbeads commented:
> > Got a few issues with this.
> > 
> > * Not sure anyone actually Asked for this change. I play forest brawler on occasion and I enjoy the class, if I wanted to dual wield I would play infantry
> 
> The host has, actually
> 

That doesnt change the fact that they will just become an off colour infantry with this, unless someone wants to code in duel wielding having actual effects

People including myself that play Forest Brawler actually enjoy it how it is

GaleZicl commented:
And people who play against Forest brawler do not enjoy it.

assortedbeads commented:
> And people who play against Forest brawler do not enjoy it.

If you lose to a speed gimped forest brawler that is entirely on you

ghkiha commented:
Cleaned all references of 'Berserker' to now be 'Reaver'

ghkiha commented:
Updated to properly emulate the 'berserker' identity. Removed dodge expert in trade for pain stun resistance. Also gave warden a better axe than his berserker underling in the form of footman axe.

grungussuss commented on code/modules/jobs/job_types/garrison/forestguard.dm:
```diff
@@ -124,41 +124,42 @@
-	tutorial = "In the goblin wars you took an oath to never wield a weapon, you just enjoy getting your hands dirty too much..."
+	name = "Forest Reaver"
+	tutorial = "In the goblin wars you took an oath to never shy from a hit, you just enjoy the chaos of battle..."
 	outfit = /datum/outfit/job/forestguard/brawler
```
outfit still brawler

ghkiha replied:
Can I change the end there and it not break anything if I just replace all brawler instances with reaver?

grungussuss commented on code/modules/jobs/job_types/garrison/forestguard.dm:
```diff
@@ -124,41 +124,42 @@
 	outfit = /datum/outfit/job/forestguard/brawler
 	category_tags = list(CTAG_FORGARRISON)
 
 /datum/outfit/job/forestguard/brawler/pre_equip(mob/living/carbon/human/H)
```
still brawler

grungussuss commented on code/modules/jobs/job_types/garrison/forestwarden.dm:
```diff
@@ -43,7 +43,7 @@
 	beltl = /obj/item/weapon/axe/iron
 	beltr = /obj/item/storage/belt/pouch/coins/rich
-	backr = /obj/item/weapon/polearm/halberd/bardiche/woodcutter
+	backr = /obj/item/weapon/polearm/halberd/bardiche/warcutter
```
what's this sneaking in?

ghkiha replied:
I documented it

ghkiha replied:
@ghkiha
[Weapons update](https://github.com/Vanderlin-Tales-Of-Wine/Vanderlin/pull/1768/commits/ef3d91decc2fe465425997b115aaa97cb361dff4) 
[ef3d91d](https://github.com/Vanderlin-Tales-Of-Wine/Vanderlin/pull/1768/commits/ef3d91decc2fe465425997b115aaa97cb361dff4)
Gives Warden a weapon befitting him and gives a weaker version to the Reaver. Footman axe to the Warden and Woodcutter to the Reaver.
